tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Checkboxes', () => {
+    test.beforeEach( async({page}) => {
+        await page.goto('/')
+        //1-2. Opening dropdown list and select 'All' option then validating 'Veterinarians' page is opened
+        await page.getByRole('button', {name: "Veterinarians"}).click()
+        await page.getByRole('link', {name: 'All'}).click()
+        await expect(page.locator('h2')).toHaveText('Veterinarians')
+    })
+
+    test('Test Case 1: Validate selected specialties', async ({page}) => {
+        //3. Locating targeted row and clicking 'Edit'
+        const tableRows = page.getByRole('row', { name: 'Helen Leary' })
```

Comment:
It's not a `tableRows`. It's a signle row for Helen Leary. Also this constant is not needed at all since it was used just once in the test in the next line. So you could just write a single like to select that Edit Vet button